app/parse.py

```diff
@@ -8,5 +10,45 @@
     duration: str
 
 
+def get_soup(url):
+    content = requests.get(url).content
+    soup = BeautifulSoup(content, "html.parser")
+    return soup
+
+
+def get_data_from_soup(soup):
+    all_courses = soup.select("div.ProfessionCard_cardWrapper__JQBNJ")
+    return [
+        {
+            "name": course.select_one("a.typography_landingH3__vTjok > h3").text,
+            "short_description": course.select_one(
+                "div.ProfessionCard_cardWrapper__JQBNJ > "
+                "p.typography_landingTextMain__Rc8BD"
+            ).text,
+            "duration": (
+                course.select_one(
+                    "div.ProfessionCard_professionTags__kCD1H > p:last-child"
+                ).text
```

Comment:
Ensure that the CSS selectors used here match the actual structure of the webpage you are scraping. If the structure changes, these selectors might not work as expected, leading to errors or incorrect data extraction.